code/testapp/src/main/java/com/adobe/edge/media/testapp/MediaTestApp.java

```diff
@@ -21,7 +21,7 @@
 import com.adobe.marketing.mobile.edge.media.Media;
 import java.util.Arrays;
 
-public class MediaAnalyticsTestApp extends Application {
+public class MediaTestApp extends Application {
```

Comment:
nit: `TestApp` generic name

Reply:
In this case I don't think we need to make the application class name generic. The Edge test app does use `TestApplication` but for Consent we use `ConsentTestApplication` and for Identity it's `EdgeIdentityApplication`. Unless you feel strongly to change the name to something like `EdgeMediaTestApplication`, I'm inclined to leave the class name as is.